1.10/Dockerfile.rhel7

```diff
@@ -32,7 +32,10 @@ LABEL summary="$SUMMARY" \
 ENV NGINX_CONFIGURATION_PATH=/opt/app-root/etc/nginx.d
 ENV NGINX_DEFAULT_CONF_PATH=/opt/app-root/etc/nginx.default.d
 
-RUN yum install -y yum-utils && \
+# To use subscription inside container yum command has to be run first (before yum-config-manager)
+# https://access.redhat.com/solutions/1443553
+RUN yum repolist > /dev/null && \
```

Comment:
In my case all that first `yum` command did was to populate the `redhat.repo` file which was until then empty. However, it seems that subsequent actions with that .repo file do not seem to affect anything other than the file itself. Which is why I though cleaning the cache might fix something but I have not expected the `yum clean` command to revert the .repo file to its previous state (`rhel-server-rhscl-7-rpms` got disabled)